packages/design-system/.storybook/main.js

```diff
@@ -38,7 +38,12 @@ module.exports = {
 	staticDirs: ['../static'],
 	addons: [
 		'@storybook/addon-a11y',
-		'@storybook/addon-essentials',
+		{
+			name: '@storybook/addon-essentials',
+			options: {
+				backgrounds: false,
```

Comment:
Since we manage the dark mode through a plugin below, we can disable it from essentials Storybook plugin